frontend_tests/node_tests/narrow_state.js

```diff
@@ -128,23 +132,26 @@ function set_filter(operators) {
 (function test_set_compose_defaults() {
     set_filter([['stream', 'Foo'], ['topic', 'Bar']]);
 
-    var opts = {};
-    narrow_state.set_compose_defaults(opts);
-    assert.equal(opts.stream, 'Foo');
-    assert.equal(opts.subject, 'Bar');
+    var stream_and_subject = narrow_state.set_compose_defaults();
+    assert.equal(stream_and_subject.stream, 'Foo');
+    assert.equal(stream_and_subject.subject, 'Bar');
 
     set_filter([['pm-with', 'foo@bar.com']]);
-    narrow_state.set_compose_defaults(opts);
-    assert.equal(opts.private_message_recipient, 'foo@bar.com');
+    var pm_test = narrow_state.set_compose_defaults();
+    assert.equal(pm_test.private_message_recipient, 'foo@bar.com');
+
+    // no real test if it covers the line it works
+    set_filter([['topic', 'duplicate'], ['topic', 'duplicate']]);
+    narrow_state.set_compose_defaults();
```

Comment:
You should assert something here. Like `assert.deepEqual(narrow_state.set_compose_defaults(), {});`.

Reply:
(Also, make sure to delete the comment in the line above)